_harp/_data.json

```diff
@@ -14,8 +14,8 @@
   "./eclipse_lyo/eclipse-lyo" : {
     "title": "Tools & SDKs for Java Developers - Eclipse Lyo"
   },
-  "sample-applications-and-code": {
-    "title": "Sample applications and code"
+  "tutorials-and-sample-applications": {
```

Comment:
aghh, this will break links and kick us off the (already low) positions in Google Search...